docs/infrastructure/metrics-explorer.asciidoc

```diff
@@ -0,0 +1,59 @@
+[role="xpack"]
+[[metrics-explorer]]
+== Using the Metrics explorer
+
+The metrics explorer allows you to create system load charts for each host you're monitoring with Metricbeat.
+
+[role="screenshot"]
+image::infrastructure/images/metrics-explorer-screen.png[Metrics Explorer in Kibana]
+
+[float]
+[[metrics-explorer-requirements]]
+=== Metrics explorer requirements and considerations
```

Comment:
Moved reqs/considerations up to the top, I think it makes more sense to outline them first